test_scripts/API/VehicleData/GpsShiftSupport/commonGpsShift.lua

```diff
@@ -0,0 +1,105 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+ --[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local utils = require('user_modules/utils')
+local commonRC = require("test_scripts/RC/commonRC")
+
+--[[ Variables ]]
+local m = actions
+m.cloneTable = utils.cloneTable
+m.radioData = commonRC.getModuleControlData("RADIO")
+m.shiftValue = {
+    true,
+    false
+}
+
+m.gpsParams = {
+    longitudeDegrees = 42.5,
+    latitudeDegrees = -83.3,
+    utcYear = 2013,
+    utcMonth = 2,
+    utcDay = 14,
+    utcHours = 13,
+    utcMinutes = 16,
+    utcSeconds = 54,
+    compassDirection = "SOUTHWEST",
+    pdop = 8.4,
+    hdop = 5.9,
+    vdop = 3.2,
+    actual = false,
+    satellites = 8,
+    dimension = "2D",
+    altitude = 7.7,
+    heading = 173.99,
+    speed =  2.78
+}
+
+--[[ Functions ]]
+function m.pTUpdateFunc(tbl)
+    tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.fullAppID].groups = {"Base-4", "Location-1"}
+end
+
+function m.getVehicleData(pShiftValue)
+    m.gpsParams.shifted = pShiftValue
+    local cid = m.getMobileSession():SendRPC("GetVehicleData", { gps = true })
+    EXPECT_HMICALL("VehicleInfo.GetVehicleData", { gps = true })
+    :Do(function(_, data)
+        m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", { gps = m.gpsParams })
+    end)
+    m.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS", gps = m.gpsParams })
```

Comment:
@TMyza In case of missing `shifted` parameter we have to check explicitly that it's not transferred to mobile